roxie/ccd/ccdserver.cpp

```diff
@@ -284,6 +284,14 @@ class IndirectAgentContext : implements IRoxieAgentContext, public CInterface
     {
         return ctx->getClientHeaders();
     }
+    virtual IProperties * getSpanContext() const override
+    {
+        return ctx->getSpanContext();
+    }
+    virtual void setSpanAttribute(const char *name, const char *value) const override
+    {
+        return ctx->setSpanAttribute(name, value);
```

Comment:
observation: This did not use to be legal (return with a void expression), but it appears it is now.